test/common/integration/rtds_integration_test.cc

```diff
@@ -34,25 +44,22 @@ envoy::config::bootstrap::v3::LayeredRuntime layeredRuntimeConfig(const std::str
             set_node_on_first_message_only: true
     - name: some_admin_layer
       admin_layer: {{}}
-  )EOF",
+)EOF",
                                        api_type, XDS_CLUSTER);
```

Comment:
This function should just set dynamic parameters, and the yaml will be stored in config.cc

params are api_type, XDS_CLUSTER, and initial_fetch_timeout.